### PR TITLE
FIx #1559 Lux Meter/Barometer configuration titles fixed

### LINKED ISSUE #1559: Title of LuxMeter/ Barometer configuration screen should be "LuxMeter/ Barometer configuration"
**Actual Behaviour**

Title of LuxMeter/ BaroMeter configuration  screen is "LuxMeter/Barometer", which is inconsistent with accelerometer

**Expected Behaviour**

Title should be "Luxmeter Configurations" and "Barometer Configurations"

**Steps to reproduce it**

Open configurations of Luxmeter or barometer

**Screenshots of the issue**

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by neel1998**
Fixes #1559  

**Changes**: 
String sent to settingsActivity with intent changed

**Screenshot/s for the changes**:

screenshot_20190220-141517
screenshot_20190220-141504

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

**APK for testing**: 
[configuration_title.zip](https://github.com/fossasia/pslab-android/files/2883749/configuration_title.zip)

### CODE REVIEW

File: app/src/main/java/io/pslab/models/PSLabSensor.java [hunk 1]
```diff
@@ -88,8 +88,10 @@
     public final String DATA_BLOCK = "data_block";
 
     public static final String LUXMETER = "Lux Meter";
+    public static final String LUXMETER_CONFIGURATIONS = "Lux Meter Configurations";
```
Comment (harsh-2711) [inline app/src/main/java/io/pslab/models/PSLabSensor.java]: Do we need this line in `PSLabSensor` file? I can't see its use anywhere

File: app/src/main/java/io/pslab/models/PSLabSensor.java [hunk 2]
```diff
@@ -88,8 +88,10 @@
...
     public static final String LUXMETER_DATA_FORMAT = "%.2f";
     public static final String BAROMETER = "Barometer";
+    public static final String BAROMETER_CONFIGURATIONS = "Barometer Configurations";
```
Comment (harsh-2711) [inline app/src/main/java/io/pslab/models/PSLabSensor.java]: This one too.

File: app/src/main/java/io/pslab/models/PSLabSensor.java (hunk 1 above)
Comment (CloudyPadmal) [reply app/src/main/java/io/pslab/models/PSLabSensor.java]: Those are for the switch cases :)

File: app/src/main/java/io/pslab/models/PSLabSensor.java [hunk 3]
```diff
@@ -88,8 +88,10 @@
     public final String DATA_BLOCK = "data_block";
 
     public static final String LUXMETER = "Lux Meter";
```
Comment (abhinavraj23) [inline app/src/main/java/io/pslab/models/PSLabSensor.java]: @neel1998 I guess then you won't be requiring this line

File: app/src/main/java/io/pslab/models/PSLabSensor.java [hunk 4]
```diff
@@ -88,8 +88,10 @@
...
+    public static final String LUXMETER_CONFIGURATIONS = "Lux Meter Configurations";
     public static final String LUXMETER_DATA_FORMAT = "%.2f";
     public static final String BAROMETER = "Barometer";
```
Comment (abhinavraj23) [inline app/src/main/java/io/pslab/models/PSLabSensor.java]: This line can also be deleted

File: app/src/main/java/io/pslab/models/PSLabSensor.java (hunk 1 above)
Comment (harsh-2711) [reply app/src/main/java/io/pslab/models/PSLabSensor.java]: Oh! Just missed that..

**Pull request merged**